reana_cluster/cli/__init__.py

```diff
@@ -59,11 +59,17 @@ def __init__(self):
 @click.option(
     '--cephfs', is_flag=True,
     help='Set cephfs volume for cluster storage.')
+@click.option(
+    '--cephfs-size',
+    type=int,
+    default=200,
```

Comment:
What do you think about also reading the value written inside the REANA-Cluster spec (`reana_cluster/configurations/reana-cluster[dev|latest].yaml`) so we have: CLI option as higest priority, then fallback to the RC spec, and if none of then is set use this default.

Reply:
Do you mean introducing new variable `CephFS_size` in `reana_cluster/configurations/reana-cluster[dev|latest].yaml`?

Reply:
Yes, it could be useful, what do you think? (it can be all lower case) I am not worried about REANA-Cluster since it is not the most user-friendly tool we have, but having a list of all needed configuration variables for deploying REANA will help to port this deployment to Helm (this cluster specs will become [values files in Helm](https://docs.helm.sh/chart_template_guide/#values-files).

Reply:
sure, sounds good, but I guess I should include it only in reana-cluster.yaml, right?